eng/pipelines/coreclr/templates/xplat-job.yml

```
@@ -93,10 +94,7 @@ jobs:
 
     - name: crossArg
       value: ''
-    - ${{ if ne(parameters.crossrootfsDir, '') }}:
-      - name: crossArg
-        value: '-cross'
-    - ${{ if and(eq(parameters.osGroup, 'OSX'), eq(parameters.archType, 'arm64')) }}:
+    - ${{ if eq(parameters.crossBuild, 'true') }}:
```

since this is a boolean it is better to do the comparison boolean vs boolean to avoid string comparisons and get more accurate results.
```suggestion
    - ${{ if eq(parameters.crossBuild, true) }}:
```